src/elements/time-input/time-input.ts

```diff
@@ -95,23 +95,33 @@ class SbbTimeInputElement extends LitElement {
     super.connectedCallback();
 
     this._findInputElement();
-    if (this._inputElement) {
-      this._updateValue(this._inputElement.value);
-    }
   }
 
   public override disconnectedCallback(): void {
     super.disconnectedCallback();
     this._abortController?.abort();
   }
 
+  public override firstUpdated(_changedProperties: PropertyValues): void {
+    super.firstUpdated(_changedProperties);
+
+    this._findInputElement();
+  }
+
   private _findInputElement(): void {
```

Comment:
nitpicky: maybe method needs renaming

```suggestion
  private _configureInputElement(): void {
```

```suggestion
  private _findAndConfigureInputElement(): void {
```

or something else :P